app/utils/fetcher.ts

```diff
@@ -0,0 +1,17 @@
+interface Headers {
+  [key: string]: string;
+}
+
+export default function fetcher(url: string, headers: Headers = {}) {
```

Comment:
If you weren't already looking, here are some good ideas for return types here, especially if you type this with `swr`'s `Fetcher` type.

https://swr.vercel.app/docs/typescript

Reply:
thanks for calling this out! I updated to use a generic to specify the return type. Let me know what you think, @rosschapman